RevenueCatUI/Templates/Template4View.swift

```diff
@@ -374,7 +374,7 @@ private struct PackageButton: View {
                     : self.configuration.colors.accent2Color
                 )
 
-            Text(Localization.localized(discount: discount, locale: self.locale))
+            Text(badge)
                 .textCase(.uppercase)
                 .foregroundColor(
                     self.selected
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 389]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Same here, maybe this should be `discountLabelHeight`<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/RevenueCat/purchases-ios/4033?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->